configure.php

```diff
@@ -198,13 +234,13 @@ function delete_files( string|array $paths ) {
 
 $author_name = ask(
 	question: 'Author name?',
-	default: run( 'git config user.name' ),
+	default: run( 'git config user.name' ) ?: ( $args['author_name'] ?? '' ),
 	allow_empty: false,
 );
 
 $author_email = ask(
 	question: 'Author email?',
-	default: run( 'git config user.email' ),
+	default: run( 'git config user.email' ) ?: ( $args['author_email'] ?? '' ),
```

Comment:
Same comment above. I'd also recommend adding a check below for an empty string for `$author_email` and bailing if so to avoid an invalid `composer.json`.

Reply:
Never mind. I see this logic is in the other PR. 🎉